hooks/price/types.go

```diff
@@ -1,30 +1,10 @@
 package price
```

Comment:
Cam we delete this file?

Reply:
Actually, `Input` and `Output` are used in `AfterEndBlock` of `hooks/price/price.go` to decode result's calldata and result.

Reply:
I see